common/cond_handle.h

```diff
@@ -73,6 +73,23 @@ static inline int cond_handle_wait(cond_handle_t* const cond,
   return pthread_cond_wait(cond, lock);
 }
 
+/**
```

Comment:
same here, can we move the documentation to somewhere else so we don't have to redefine each time?